src/HydratorAwareTrait.php

```diff
@@ -1,46 +1,35 @@
 <?php
 /**
- * Zend Framework (http://framework.zend.com/)
- *
- * @link      http://github.com/zendframework/zf2 for the canonical source repository
- * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
- * @license   http://framework.zend.com/license/new-bsd New BSD License
+ * @see       https://github.com/zendframework/zend-hydrator for the canonical source repository
+ * @copyright Copyright (c) 2010-2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-hydrator/blob/master/LICENSE.md New BSD License
  */
 
+declare(strict_types=1);
+
 namespace Zend\Hydrator;
 
 trait HydratorAwareTrait
 {
     /**
      * Hydrator instance
      *
-     * @var HydratorInterface
-     * @access protected
+     * @var null|HydratorInterface
      */
     protected $hydrator = null;
```

Comment:
setting `null` as default value is redundant, we have it also in several other places and in all of them it can be removed